projects/step-frontend/src/lib/modules/timeseries/performance-view/performance-view.component.ts

```diff
@@ -568,13 +579,14 @@ export class PerformanceViewComponent implements OnInit, OnDestroy {
     return this.tableChart.refresh(clonedRequest); // refresh the table
   }
 
-  handleRangeReset(newRange: TSTimeRange) {
-    this.responseTimeChart.resetZoom();
-    this.byStatusChart.resetZoom();
-    this.byStatusChart.resetZoom();
-    this.summaryChart.resetZoom();
-    this.throughputChart.resetZoom();
-    this.updateTable();
+  handleZoomReset() {
+    // TODO uncomment these when the uplot sync feature is removed
+    // this.responseTimeChart.resetZoom();
+    // this.byStatusChart.resetZoom();
+    // this.byStatusChart.resetZoom();
+    // this.summaryChart.resetZoom();
+    // this.throughputChart.resetZoom();
```

Comment:
Will this all work still without these commented out lines? What is the plan for th euplot sync feature to be removed?